packages/core/src/tracing/instrumentation/cloud/aws/dynamodb.js

```diff
@@ -0,0 +1,179 @@
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2021
+ */
+
+'use strict';
+
+const shimmer = require('shimmer');
+const requireHook = require('../../../../util/requireHook');
+const cls = require('../../../cls');
+const { EXIT, isExitSpan } = require('../../../constants');
+const tracingUtil = require('../../../tracingUtil');
+
+const operationsInfo = {
+  createTable: { op: 'create' },
+  // deleteTable: { op: 'delete' },
```

Comment:
What is the reason for leaving that one out?

Reply:
> What is the reason for leaving that one out?

`deleteTable` is not instrumented in Java or Go instrumentations. There is a discussion that started on Slack about this topic. We will bring it to the Dev Tracer meeting ASAP.

Reply:
Ah, so for consistency. Gotcha, that makes sense.